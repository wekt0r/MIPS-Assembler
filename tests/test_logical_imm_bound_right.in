xori $s0, $s1, 40000
andi $s0, $s1, 65534
ori $s1, $s2, 50000
