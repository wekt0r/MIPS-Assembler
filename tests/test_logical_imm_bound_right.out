.text
00000000	3A309C40	xori	$s0,$s1,40000
00000004	3230FFFE	andi	$s0,$s1,65534
00000008	3651C350	ori	$s1,$s2,50000
